ci/run_tests.sh

```diff
@@ -8,14 +8,20 @@ source "${CI_DIR}/common/build.sh"
 source "${CI_DIR}/common/test.sh"
 source "${CI_DIR}/common/suite.sh"
 
+# Build.
 enter_suite build
```

Comment:
isn't "enter suite ..." already descriptive enough?

Reply:
I found it hard to visualize the different sections therein - so it is more meant as a section headline.